sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdk.java

```diff
@@ -88,7 +95,24 @@ public void recordSpanData(SpanData span) {}
    *
    * <p>After this is called all the newly created {@code Span}s will be no-op.
    */
-  public void shutdown() {}
+  public void shutdown() {
+    if (isStopped) {
+      logger.log(Level.WARNING, "Calling shutdown() multiple times.");
+      return;
+    }
+    synchronized (this) {
+      for (SpanProcessor spanProcessor : registeredSpanProcessors) {
```

Comment:
the activeSpanProcessor is a MultiSpanProcessor to avoid using for loops everywhere.

Reply:
That's true, fixed.